src/style.css

```diff
@@ -1,3 +1,72 @@
 body {
   background-color: bisque;
+  font-size: 1rem;
+  font-family: "Sora", sans-serif;
+}
```

Comment:
- [ ] Kindly aligny the submit button according to the provided design

Given design
![image](https://user-images.githubusercontent.com/81932478/202872121-f0c4043c-385b-4033-89fe-4e3b8489a4a6.png)

Your design
![image](https://user-images.githubusercontent.com/81932478/202872148-4599097c-e997-44e3-b179-87270a35d305.png)